.drone.yml

```diff
@@ -4086,6 +4086,11 @@ clone:
   disable: true
 
 steps:
+  - name: Check if commit is tagged
```

Comment:
Will `dronegen` wipe your your .drone.yaml changes the next time it is run?

Reply:
`dronegen` doesn't handle the `promote-build` pipeline -- has a pass through for "unmigrated" piplelines.  See:

https://github.com/gravitational/teleport/blob/8db6aa58833ca5af93f0cdcae56b4dd862c81dfe/dronegen/main.go#L51-L108

A way to check this is compare the comment at the top of different pipelines.  Compare:

https://github.com/gravitational/teleport/blob/8db6aa58833ca5af93f0cdcae56b4dd862c81dfe/.drone.yml#L414-L422

vs

https://github.com/gravitational/teleport/blob/8db6aa58833ca5af93f0cdcae56b4dd862c81dfe/.drone.yml#L4795-L4797